services/couchdb/Dockerfile

```diff
@@ -1 +1 @@
-FROM klaemo/couchdb-ssl:latest
+FROM klaemo/couchdb:latest
```

Comment:
Why don't we use the [couchdb official image](https://hub.docker.com/_/couchdb/)? (seems to be fairly new given the number of downloads vs. the number of downloads of some community image)